source/reference/mongosync.txt

```diff
@@ -1,9 +1,95 @@
 .. _c2c-mongosync:
 
+.. default-domain:: mongodb
+
+.. program:: mongosync
+
 =============
 ``mongosync``
 =============
 
-.. default-domain:: mongodb
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Synopsis
+--------
+
+The ``mongosync`` binary is the primary process used in Cluster to Cluster Sync.
+It migrates data from one cluster to another and can keep the clusters
+in continuous sync.
+
+This document provides a complete overview of all command-line options.
+
+Options
+-------
+
+Global Options
+~~~~~~~~~~~~~~
+
+.. option:: --cluster0 <URI>
+
+   Sets the URI of the first cluster.  The first cluster can serve as either the
```

Comment:
It's probably a good idea to mention that source and destination have to be MongoDB v6.0

Reply:
Fixed